charts/sonarqube-dce/values.yaml

```diff
@@ -570,7 +570,7 @@ networkPolicy:
   # additionalNetworkPolicies:
 
 initContainers:
-  # image: busybox:1.36
+  # image: ubuntu:24.04
```

Comment:
We should set this value not to be `ubuntu` (even if commented). This might be a wrong suggestion, as the default value is the usual `sonarqube`. I would just leave `# image: `.

Reply:
hmm i agree, the goal there was to show you can change the image, but it should be explicit that it is not required.
I'll add the ubuntu above the image in a dedicated comment.